packages/utils/src/lib/auth/siwe.spec.ts

```
@@ -5,6 +5,7 @@ import { Url } from '../url';
 import { createWalletClient, http } from 'viem';
 import { sepolia } from 'viem/chains';
 import { privateKeyToAccount } from 'viem/accounts';
+import { verify } from 'crypto';
```

ESLint

[error] 'verify' is defined but never used.

(@typescript-eslint/no-unused-vars)